src/smartMirrorProject/radarWidget/static/radarWidget/style.css

```diff
@@ -0,0 +1,11 @@
+.radar-widget {
+    max-height: 200px;
```

Comment:
Waarom staat hier max 200px terwijl er in weather-radar.html 256 staat

Reply:
Foutje, opgelost